docker/testcafe-docker.sh

```diff
@@ -6,4 +6,4 @@ dbus-daemon --session --fork
 Xvfb :1 -screen 0 "${XVFB_SCREEN_WIDTH}x${XVFB_SCREEN_HEIGHT}x24" >/dev/null 2>&1 &
 export DISPLAY=:1.0
 fluxbox >/dev/null 2>&1 &
-node /usr/lib/node_modules/testcafe/bin/testcafe-with-v8-flag-filter.js --ports 1337,1338 "$@"
+node /usr/lib/node_modules/testcafe/bin/testcafe.js --ports 1337,1338 "$@"
```

Comment:
Remove the `testcafe-with-v8-flag-filter.js` file.

Reply:
but it's removed

Reply:
Sorry, I've missed this.